environment-docs.yml

```diff
@@ -4,8 +4,13 @@ channels:
 - defaults
 dependencies:
 - python=3.7
-- pywps>=4.2
+- pywps>=4.2.4
 - sphinx
 - nbsphinx
 - ipython
+- ipywidgets
+- pip   
+- pip:
+  - birdhouse-birdy
```

Comment:
`birdy` is in Conda, no need to use `birdhouse-birdy` via Pip.

Reply:
I think `environment-docs.yml` is for ReadTheDocs build, then `birdy` is probably not even required.